### DISCUSSION TITLE
clean-detailed theme

### DISCUSSION THREAD

**Discussion by nutanixguy** [JanDeDobbeleer/oh-my-posh#2935] [Troubleshoot]
When the clean-detailed theme is applied on macos using zsh the commands show up above the prompt is that by design? Thank you in advance 

![CleanShot 2022-10-12 at 09 54 25](https://user-images.githubusercontent.com/30190939/195361928-d9ecdffe-acdc-4c1f-beba-30c83a231100.png)

**Comment by JanDeDobbeleer** [comment]
@jamierterrell this is the [transient prompt](https://ohmyposh.dev/docs/configuration/transient) in effect.

**Reply by nutanixguy** [reply]
Thank you, sorry just getting started with the app, figured it was something simple.
